### PR TITLE
fixed cy.intercepts

### PR DESCRIPTION
**Pull request by dtaylor113**

### CODE REVIEW

File: frontend/packages/operator-lifecycle-manager/integration-tests-cypress/tests/operator-uninstall.spec.ts [hunk 1]
```diff
@@ -91,12 +91,14 @@ describe(`Testing uninstall of ${testOperator.name} Operator`, () => {
...
     // letting request go thru unchanged and mocking the response
-    cy.intercept('DELETE', '/api/kubernetes/apis/operators.coreos.com/*', (req) => {
-      // examples:
-      // .../api/kubernetes/apis/operators.coreos.com/v1alpha1/namespaces/test-mpnsw/subscriptions/datagrid-foobar
-      // .../api/kubernetes/apis/operators.coreos.com/v1alpha1/namespaces/test-mpnsw/clusterserviceversions/datagrid-operator.v8.2.0-foobar
-      req.url = `${req.url}-foobar`;
-    }).as('deleteOperatorSubscriptionAndCSV');
+    cy.intercept(
+      'DELETE',
+      '/api/kubernetes/apis/operators.coreos.com/*/namespaces/*/*/*',
```
Comment (spadgett) [inline frontend/packages/operator-lifecycle-manager/integration-tests-cypress/tests/operator-uninstall.spec.ts]: I'm debating if this much specificity in the URL is better or worse. Was there a problem with `/api/kubernetes/apis/operators.coreos.com/*` or did you just want something more precise?

Comment (dtaylor113) [reply frontend/packages/operator-lifecycle-manager/integration-tests-cypress/tests/operator-uninstall.spec.ts]: I was having issues with the wildcard, seemed like dir breaks (`/`) need to be specified!?  ...let me try it again
...yeah, changed it to `/api/kubernetes/apis/operators.coreos.com/*/namespaces/*` and it doesn't work.

[...requests for paths in nested directories may be affected](https://docs.cypress.io/guides/references/migration-guide#URL-matching-is-stricter)

Comment (spadgett) [reply frontend/packages/operator-lifecycle-manager/integration-tests-cypress/tests/operator-uninstall.spec.ts]: @dtaylor113 We should be able to use `/api/kubernetes/apis/operators.coreos.com/**`

https://docs.cypress.io/api/commands/intercept#Usage

Comment (spadgett) [reply frontend/packages/operator-lifecycle-manager/integration-tests-cypress/tests/operator-uninstall.spec.ts]: Note that it needs two `**`

Comment (dtaylor113) [reply frontend/packages/operator-lifecycle-manager/integration-tests-cypress/tests/operator-uninstall.spec.ts]: thanks, that worked.  Updated this PR.

**Review by spadgett — APPROVED** [review]

**Pull request merged**